demo/WebexMeetingsWidgetDemo.jsx

```diff
@@ -14,10 +18,19 @@ export default function WebexMeetingsWidgetDemo({token, fedramp}) {
   && navigator.userAgent.includes('iPhone OS 15_1');
 
   const displayButtonEnabled = token && destination && !displayWidget;
+  const spaceIDError = `Using the space ID as a destination is no longer supported. Please refer this <a href="https://github.com/webex/webex-js-sdk/wiki/Migration-guide-for-USM-meeting" target="_blank">documentation</a> to migrate to use the meeting ID or SIP address.`;
+  let DisplayMeetingWidgetStatusElm = document.querySelector('#display-meeting-widget-status');
 
   const handleDisplayMeetingWidget = (event) => {
     event.preventDefault();
-    setDisplayWidget(true);
+    const hydraId = getHydraId(destination);
+    if(!hydraId.room) {
+      setDisplayWidget(true);
+      DisplayMeetingWidgetStatusElm.innerHTML = '';
+    } else {
+      DisplayMeetingWidgetStatusElm.innerHTML = spaceIDError;
```

Comment:
Instead of setting the error message like this, we can do conditional rendering. There is no need to find the dom element.

Reply:
@rarajes2, I have added some comments on the code for more clarity, Also we are conditionally rendering content inside the DisplayMeetingWidgetStatusElm element. If the condition is met, we set the innerHTML of this to the desired content (in this case, the error message spaceIDError).

Reply:
> There is no need to find the dom element.

I meant we can simply render the variable with an error message inside the html element instead of finding the element from dom using query selector and then updating the innerHTML. React will take care of re-rendering once the value of the variable updates.

Reply:
@rarajes2, thanks, let me have a look and update it.

Reply:
@rarajes2 Addressed. Please check..